tests/scripts/local_monitoring_tests/Local_monitoring_tests.md

```diff
@@ -91,6 +91,30 @@ Here are the test scenarios:
 - Test with invalid values such as blank, null or an invalid value for name query parameter in listMetadataProfiles API
 - List metadata profiles without creating metadata profile
 
+### **Update Metadata Profile API tests**
+
+Here are the test scenarios:
+
+- Update a metadata profile with a valid json using the API, list and validate the metadata profiles before and after 
```

Comment:
Please check if all the scenarios are covered here & updated the expected behaviour